// // the try statement allow you to define a block of code to be tested for errors while it is being executed.

// // the catch statement allow you to define a block of code to be executed, if an error occurs in the try block.
// try{
//     console.log(a);
// }
// catch(error){
//     console.log("The error is handled");
//     console.log(error);
// }